### PR TITLE
Refactor Wigner function into module

### PR DESCRIPTION
**Pull request by sduquemesa**
**Context:**
Calculation of the Wigner function is already included in Mr Mustard but hidden in the graphics module

**Description of the Change:**
This PR moves the Wigner function calculation to its own module and numbifies it.

**Benefits:**
Now you can calculate the Wigner function using
```python
from mrmustard.utils.wigner import wigner_discretized

wigner_discretized(dm, q, p) # dm is a density matrix
```

It should be faster as well because it uses numba jit.

**Possible Drawbacks:**
None

### CODE REVIEW

**Comment by ziofil** [comment]
Nice! We can easily write the P and Q as well right?

**Comment by sduquemesa** [comment]
Correct! Though I think all this functions will have a better place to be in soon with the representations. Right now it feels a bit missplaced.

File: mrmustard/utils/wigner.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        array: array containing the discretized Wigner function
+    """
+
+    Q = np.outer(pvec, np.ones_like(qvec))
+    P = np.outer(np.ones_like(pvec), qvec)
+    cutoff = rho.shape[-1]
+    A = (Q + P * 1.0j) / (2 * np.sqrt(hbar / 2))
+
+    cutoff = rho.shape[-1]
+    A = (Q + P * 1.0j) / (2 * np.sqrt(hbar / 2))
```
Comment (elib20) [inline mrmustard/utils/wigner.py:44]: Repeated lines?

**Review by elib20 — APPROVED** [review]
Looks good! Why not replace the plot_wigner function in graphics with the new function?

File: mrmustard/utils/wigner.py (hunk 1 above)
Comment (sduquemesa) [reply mrmustard/utils/wigner.py:44]: Indeed. Good catch!
```suggestion
```

**Pull request merged**